bin/locals

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+$:.unshift(File.expand_path("../lib", __dir__))
+require "yarp"
+
+source = ARGV[0] == "-e" ? ARGV[1] : File.read(ARGV[0])
+
+puts "CRuby:"
+p YARP.const_get(:Debug).cruby_locals(source)
```

Comment:
Why a `const_get` here instead of `YARP::Debug`

Reply:
`Debug` is a private constant, just to make it clear it's only meant for internal testing. It also means we don't impose on JRuby/TruffleRuby to implement these interfaces.